dashboards-observability/.cypress/integration/event_analytics.spec.js

```diff
@@ -400,7 +400,6 @@ describe('Switch on and off livetail', () => {
     cy.wait(delay * 2);
     cy.get('.euiToastHeader__title').contains('On').should('exist');
 
-    cy.get('[data-test-subj=eventLiveTail]').click();
```

Comment:
Why was this removed from cypress? Don't we switch on the live tail for cypress?